src/Module/AllFriends.php

```diff
@@ -63,25 +63,22 @@ public static function content(array $parameters = [])
 		DI::page()['aside'] = "";
 		Model\Profile::load($app, "", Model\Contact::getByURL($contact["url"], false));
 
-		$total = Model\GContact::countAllFriends(local_user(), $cid);
+		$total = Model\Contact\Relation::countFollows($cid);
 
 		$pager = new Pager(DI::l10n(), DI::args()->getQueryString());
 
-		$friends = Model\GContact::allFriends(local_user(), $cid, $pager->getStart(), $pager->getItemsPerPage());
```

Comment:
Please keep using the `GContact` methods or include the extra condition they feature in this module.

Reply:
The only think in the `GContact` function is this here:

```
$cids = Contact::getPublicAndUserContacID($cid, $uid);
```

And we don't need this, since the function will always be called with the public contact id.

Reply:
Ah, fair enough, most others have filters though.